### PR TITLE
build StgMaker on master

### PR DESCRIPTION
**Pull request by jdbrice**
This allows StRoot/StgMaker to be built. 
I modified the GenFit and Kitrack dependencies to use the archive tar.gz instead of git so that git does not need to be installed as an additional container package.

In order to prevent updates as I work on StgMaker I removed StRoot/StgMaker from this repo. Instead I mount StRoot/StgMaker into the container when I build/work

### CODE REVIEW

**Comment by plexoos** [comment]
I am not sure I understand the purpose of the following two commits: b75daf3 and 0a64cfc. They seem to cancel out each other

**Comment by plexoos** [comment]
I think we should bring StgMaker to master branch by merging the existing stg-kitrack-genfit branch first. I created a pull request #9

File: cmake/target_properties.cmake [hunk 1]
```diff
@@ -66,16 +66,14 @@ if(TARGET StgMaker)
...
+			URL "https://github.com/star-bnl/KiTrack/archive/efd317b86ce2a174e50ebe183582b36102385bb1.tar.gz"
 			CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${STAR_INSTALL_PREFIX} -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
 	)
 
 	ExternalProject_Add(
 			GenFit
 			PREFIX "external/"
-			GIT_REPOSITORY "https://github.com/star-bnl/GenFit.git"
-			GIT_TAG "b496504a"
+			URL "https://github.com/star-bnl/GenFit/archive/7833bbda914ed24b420091f302ff3569537edc7e.tar.gz"
```
Comment (plexoos) [inline cmake/target_properties.cmake:76]: Last time I checked the latest version of GenFit (7833bbda as you requested) could not be compiled with ROOT5. If you confirm that it works for you I will double check my previous findings

Comment (jdbrice) [reply cmake/target_properties.cmake:76]: You are right. I  pasted the wrong hash into the URL the first time, but I thought I fixed it.

**Comment by jdbrice** [comment]
Merging in the stgmaker-kitrack-genfit branch (other pull request) is fine. This was supposed to be exactly that with:
1) remove StRoot/
2) change cmake dependencies from using git to url (with archive ter.gz)

**Pull request closed without merging**